locales/nb/validation.php

```diff
@@ -20,14 +20,14 @@
     'alpha_dash'           => ':attribute må kun bestå av bokstaver, tall og bindestreker.',
     'alpha_num'            => ':attribute må kun bestå av bokstaver og tall.',
     'array'                => ':attribute må være en matrise.',
-    'attached'             => 'This :attribute is already attached.',
+    'attached'             => ':attribute er allerede vedlagt.',
     'before'               => ':attribute må være en dato før :date.',
     'before_or_equal'      => ':attribute må være en dato før eller lik :date.',
     'between'              => [
-        'array'   => ':attribute må ha mellom :min - :max elementer.',
-        'file'    => ':attribute må være mellom :min - :max kilobytes.',
         'numeric' => ':attribute må være mellom :min - :max.',
+        'file'    => ':attribute må være mellom :min - :max kilobytes.',
         'string'  => ':attribute må være mellom :min - :max tegn.',
+        'array'   => ':attribute må ha mellom :min - :max elementer.',
```

Comment:
sort

Reply:
Same here, compared to and changed according to the English file.